### DISCUSSION TITLE
GridViewにタイトル等の情報を追加したい

### DISCUSSION THREAD

**Discussion by Stella2211** [aipictors/app#20] [アイデア - Ideas]
お題作品一覧やタグ一覧、人気作品等の画面に、タイトル・アカウント名・いいねボタンを表示させたいです。
画像のみの表示に比べて、こちらの方がより作品の魅力を伝えることができると思います。
アプリに簡易的に実装してみたものがこちらです。
GraphQLからのレスポンスにユーザー情報が含まれていないため、ユーザーアイコン・アカウント名は実装できていません。
![Simulator Screenshot - iPhone 14 Pro Max - 2023-08-19 at 19 41 19](https://github.com/aipictors/aipictors-flutter/assets/104188098/7de1bd93-4380-415f-a696-949db75124f7)

**Comment by shigurenimo** [comment]
これを実装する場合はCardに切り替える。 https://m3.material.io/components/cards/overview

**Comment by shigurenimo** [comment]
作品をよく魅せる目的であればユーザ名は不要に感じる。ただ、インスタやPixivのように画像だけでも十分な気がする。

**Comment by Stella2211** [comment]
画像のみでもいい気がしてきたので、クローズします。
